README.rst

```diff
@@ -252,7 +253,7 @@ and screen capture functionality:
 * The `luma.emulator.device.pygame` device uses the `pygame` library to
   render the displayed image to a pygame display surface.
 
-Invoke the demos with::
+*After installing* luma.emulator (see Documentation link below) you can invoke the demos with::
```

Comment:
luma.emulator isn't required to run the examples though.

Reply:
I agree, but this is in the 'Emulators' section of the readme document and the examples given are both using `--display capture`, so they won't work without luma.emulator...